app/src/main/java/org/sagebionetworks/research/mpower/tracking/model/SelectionUIFormItem.java

```diff
@@ -0,0 +1,7 @@
+package org.sagebionetworks.research.mpower.tracking.model;
+
+/**
+ * Marker to indicate that the class implementing this interface can be displayed on a SelectionFragment.
+ */
+public interface SelectionUIFormItem {
```

Comment:
Is there anything that SelectionFragment needs to display these that should go in this interface?  vars or functions?